src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/TestHelpers.cs

```diff
@@ -56,4 +59,30 @@ internal static class TestHelper
                 null
                 );
     }
+
+    //This is to make sure normalising the keys done correctly with culture invariant
+    //Lower case of SETTINGSKEY in Turkish is not settingskey but settıngskey
+    //https://en.wikipedia.org/wiki/Dotted_and_dotless_I 
+    public class TemporaryCultureSwitch : IDisposable
```

Comment:
instead of adding a new file this made more sense. To remove the confusion I've changed the filename to "TestHelpers" instead of "TestHelper"

Reply:
I'm not convinced it does make more sense; if we end up twenty test helpers then we probably wouldn't want them in all in the same class.
However, I don't feel strongly about it when there are only two.